src/main/java/org/kohsuke/github/GHOrganization.java

```diff
@@ -128,6 +128,24 @@ public GHCreateRepositoryBuilder createRepository(String name) {
                 .toIterable(GHTeam[].class, item -> item.wrapUp(this));
     }
 
+    /**
+     * Gets a single team by ID.
+     *
+     * @param id
+     *            the id
+     * @return the team
+     * @throws IOException
+     *             the io exception
+     *
+     * @see <a href= "https://developer.github.com/v3/teams/#get-team-by-name">documentation</a>
+     */
+    public GHTeam getTeam(int id) throws IOException {
+        return root.createRequest()
+                .withUrlPath(String.format("/orgs/%s/teams/%d", login, id))
```

Comment:
The reference docs say you need `org_id` with when passing `team_id`.  Please add tests showing this working correctly.

Reply:
I see that all path is different..
Fixed when fixing tests..